examples/Targeting/SearchForTargetableLanguagesAndCarriers.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Common;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+
+using System;
+
+using static Google.Ads.GoogleAds.V3.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.ProximityRadiusUnitsEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This code example illustrates how to search for language constants by name that
+    /// includes the specified keyword. Then it searches for all the available mobile carrier
+    /// constants with a given country code.
+    /// </summary>
+    public class SearchForTargetableLanguagesAndCarriers : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            SearchForTargetableLanguagesAndCarriers codeExample =
+                new SearchForTargetableLanguagesAndCarriers();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The keyword to search for.
+            string searchKeyword = "INSERT_SEARCH_KEYWORD_HERE";
```

Comment:
The golden has a [different variable name](https://github.com/googleads/google-ads-perl/pull/9/files#diff-6fc4b388e4e70ab31ee62fbb7db81a92R46)

Reply:
Done.